includes/derivatives.inc

```diff
@@ -305,14 +305,14 @@ function islandora_ocr_derive_ocr(AbstractObject $object, $force = FALSE) {
  */
 function islandora_ocr_derive_hocr(AbstractObject $object, $force = FALSE) {
   $language_array = $object->relationships->get(ISLANDORA_RELS_EXT_URI, 'hasLanguage');
-  // Logic for OCR generation from islandora_book_batch.
-  $generate_ocr = 'TRUE';
-  $batch_ocr = $object->relationships->get(ISLANDORA_RELS_EXT_URI, 'generate_ocr');
-  if ($batch_ocr) {
-    $generate_ocr = $batch_ocr[0]['object']['value'];
+  // Logic for HOCR generation from islandora_book_batch.
+  $generate_hocr = 'TRUE';
+  $batch_hocr = $object->relationships->get(ISLANDORA_RELS_EXT_URI, 'generate_hocr');
+  if ($batch_hocr) {
+    $generate_hocr = $batch_hocr[0]['object']['value'];
   }
   $language = isset($language_array[0]['object']['value']) ? $language_array[0]['object']['value'] : 'eng';
-  if ($language == 'no_ocr' || $generate_ocr == 'FALSE') {
+  if ($language == 'no_ocr' || $generate_hocr == 'FALSE') {
     $to_return['success'] = TRUE;
     $to_return['messages'][] = array(
       'message' => t('Skipped adding @dsid to @id.'),
```

Comment:
Because we have an option in the select box to not perform OCR or choose the language.
https://github.com/Islandora/islandora_ocr/blob/7.x/includes/utilities.inc#L125